<!--  <arg name="laptop" default="True" />  -->
<!--   <arg name="depth_camera" default="False" /> -->
<!--   <arg name="save_data" default="False"/> -->

<!--   <node -->
<!--     pkg="people_tracking" -->
<!--     type="depth_image.py" -->
<!--     name="depth" -->
<!--     output="screen" -->
<!--     args="$(arg laptop) $(arg depth_camera) $(arg save_data)" -->
<!--   /> -->

<launch>
  <node
    pkg="cv_camera"
    type="cv_camera_node"
    name="Webcam"
    output="screen"
  />

  <node
    pkg="people_tracking_v2"
    type="yolo.py"
    name="People_detection"
    output="screen"
  />

  <node
    pkg="people_tracking_v2"
    type="HoC.py"
    name="HoC"
    output="screen"
  />

  <node
    pkg="people_tracking_v2"
    type="pose_estimation_node.py"
    name="Pose_estimation"
    output="screen"
  />

  <!-- Uncomment these nodes when ready -->
  <!--
  <node
    pkg="people_tracking_v2"
    type="face_recognition_node.py"
    name="Face"
    output="screen"
  />
  
  -->
</launch>